tcf_website/views/profile.py

```diff
@@ -51,4 +53,21 @@ def profile(request):
 @login_required
 def reviews(request):
     """User reviews view."""
-    return render(request, 'reviews/user_reviews.html')
+    # Handled separately because it requires joining 1 more table (i.e. Vote)
+    upvote_stat = Review.objects.filter(user=request.user).aggregate(
+        total_review_upvotes=Count('vote', filter=Q(vote__value=1)),
+    )
+    # Get other statistics
+    other_stats = User.objects.filter(id=request.user.id).aggregate(
+        total_reviews_written=Count('review'),
+        average_review_rating=(
+            Avg('review__instructor_rating') +
+            Avg('review__enjoyability') +
+            Avg('review__recommendability')
+        ) / 3,
+    )
+    # Merge the two dictionaries (NOTE: can be simplified in Python 3.9)
```

Comment:
hmm any recommendations on when we should upgrade python?

Reply:
Moving the discussion to https://github.com/thecourseforum/theCourseForum2/pull/284#issuecomment-737737870